src/cljs/proton/layers/core/keybindings.cljs

```diff
@@ -162,6 +162,9 @@
              :n {:title "line numbers"
                  :target (fn [atom] (.getView (.-views atom) (.getActiveTextEditor (.-workspace atom))))
                  :action "editor:toggle-line-numbers"}
+             :h {:category "highlight"
+                 :h {:title "highlight current line"
```

Comment:
I was following the spacemacs keybinding - it has a few additional "highlight" toggles so `SPC t h h` is the keybinding for line highlighting.